### PR TITLE
java-11-openjdk: Fix parallel build

### PR DESCRIPTION
**Pull request by th0ma7**
_Motivation:_  A few left over to fix, namely rabbitmq, java and deluge...
_Linked issues:_  Follow-up on #4854

### Checklist
- [ ] Build rule `all-supported` completed successfully
- [ ] Package upgrade completed successfully
- [ ] New installation of package completed successfully

### CODE REVIEW

File: cross/java-11-openjdk/Makefile [hunk 1]
```diff
@@ -75,6 +76,8 @@ CONFIGURE_ARGS += --disable-hotspot-gtest
 # Disable all GUI related
 CONFIGURE_ARGS += --enable-headless-only
 
+COMPILE_MAKE_OPTIONS += JOBS=$(NCPUS)
```
Comment (publicarray) [inline cross/java-11-openjdk/Makefile:79]: It automagically detects how many jobs should run, so you can remove the line

**Comment by publicarray** [comment]
Thanks, I think your version looks cleaner, but I'm wondering if there should be a more obvious opt out for a package. Someone not knowing the framework would not see the difference between the default compile target `$(MAKE) -j$(NCPUS) $(COMPILE_MAKE_OPTIONS)` target and `$(MAKE) $(COMPILE_MAKE_OPTIONS)` unless they looked at the compile makefile.

I'm not sure that `$(PSTAT_TIME)` should be in the package makefile. I feel It's more for us framework developers and so package developers should not worry about that one IMHO.

File: cross/java-11-openjdk/Makefile (hunk 1 above)
Comment (th0ma7) [reply cross/java-11-openjdk/Makefile:79]: aaahhh, that I didn't know... still, i'd add it by default to be consistent but its just me.
@publicarray  and anyhow, I guess I can cancel this PR as yours superseeds it?

**Comment by th0ma7** [comment]
@publicarray really a chat engine of some sort would be best as I guess I'm now puzzled on to which PR should go in :)

**Comment by th0ma7** [comment]
Superseded by #4858 (Thnx @publicarray )

**Pull request closed without merging**